create-leo-app/template-vanilla/package.json

```diff
@@ -1,16 +1,19 @@
 {
-    "name": "aleo-starter",
-    "private": true,
-    "version": "0.0.0",
-    "type": "module",
-    "scripts": {
-        "dev": "vite",
-        "build": "vite build",
-        "preview": "vite preview"
-    },
-    "devDependencies": {
-        "@provablehq/sdk": "^0.8.0",
-        "tslib": "^2.8.1",
-        "vite": "^6.0.11"
-    }
+  "name": "aleo-starter",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "test:e2e": "sh scripts/test.sh"
```

Comment:
What is the point of having this be a script? Would prefer to get rid of this across all templates unless there's a good reason. Would just replace this with: `"test:e2e": "playwright test --headed"`